### PR TITLE
[WEBSITE-996] Updates to people, meta and homepage views

### PR DESCRIPTION
**Pull request by vinternet-pds**

### CODE REVIEW

**Review by naseberry — APPROVED** [review]

**Review by UsmanAfzal — COMMENTED** [review]
contact_points views should all be inside the section--primary as there is no heading for the list

**Review by UsmanAfzal — COMMENTED** [review]
Can we add space between every section

File: app/views/people/index.html.haml [hunk 1]
```diff
@@ -1,6 +1,11 @@
...
-
-= render partial: 'pugin/components/letter-navigation', locals: { route_symbol: :people_a_z_letter_path }
-= render partial: 'pugin/cards/person-list', locals: { people: @people, letter: params[:letter] }
+.section--primary
+  .container
+    %h1= page_title
+%section
+  .container
+    = render partial: 'pugin/components/letter-navigation', locals: { route_symbol: :people_a_z_letter_path }
+%section#content
```
Comment (UsmanAfzal) [inline app/views/people/index.html.haml:9]: add a space between sections

File: app/views/people/current_members_letters.html.haml [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
-= render partial: 'pugin/cards/person-list', locals: { people: @people, letter: params[:letter] }
-
-%p= link_to("#{t('view_all').capitalize} #{t('current_and_former')} #{t('member_plural')}", people_members_a_z_letter_path('a'))
+.section--primary
+  .container
+    %h1= page_title
+%section
+  .container
+    = render partial: 'pugin/components/letter-navigation', locals: { route_symbol: :people_members_current_a_z_letter_path }
+%section#content
```
Comment (UsmanAfzal) [inline app/views/people/current_members_letters.html.haml:9]: add a space between sections

File: app/views/contact_points/index.html.haml [hunk 3]
```diff
@@ -1,22 +1,26 @@
...
-          %dd= contact_point.phone_number
-        - if contact_point.fax_number.present?
-          %dt= "#{t('contact_points.fax')}:"
-          %dd= contact_point.fax_number
-        - contact_point.postal_addresses.each do |postal_address|
-          - if postal_address.full_address.present?
-            %dt= "#{t('contact_points.address')}:"
-            %dd= postal_address.full_address
-      %p= link_to "#{t('contact_points.add_address_book').capitalize}", contact_point_path(contact_point.graph_id), target: '_blank'
+%section
```
Comment (UsmanAfzal) [inline app/views/contact_points/index.html.haml:7]: Should all sit within section--primary, no heading exists within this section

File: app/views/people/letters.html.haml [hunk 4]
```diff
@@ -1,6 +1,11 @@
...
-
-= render partial: 'pugin/components/letter-navigation', locals: { route_symbol: :people_a_z_letter_path }
-= render partial: 'pugin/cards/person-list', locals: { people: @people, letter: params[:letter] }
+.section--primary
+  .container
+    %h1= page_title
+%section
+  .container
+    = render partial: 'pugin/components/letter-navigation', locals: { route_symbol: :people_a_z_letter_path }
+%section#content
```
Comment (UsmanAfzal) [inline app/views/people/letters.html.haml:9]: Add a space between sections

File: app/views/people/members.html.haml [hunk 5]
```diff
@@ -1,6 +1,11 @@
...
-= render partial: 'pugin/components/letter-navigation', locals: { route_symbol: :people_members_a_z_letter_path }
-= render partial: 'pugin/cards/person-list', locals: { people: @people, letter: params[:letter] }
+.section--primary
+  .container
+    %h1= page_title
+%section
+  .container
+    = render partial: 'pugin/components/letter-navigation', locals: { route_symbol: :people_members_a_z_letter_path }
+%section#content
+  .container
```
Comment (UsmanAfzal) [inline app/views/people/members.html.haml:10]: Add a space between sections

File: app/views/people/members_letters.html.haml [hunk 6]
```diff
@@ -1,6 +1,11 @@
...
-
-= render partial: 'pugin/components/letter-navigation', locals: { route_symbol: :people_members_a_z_letter_path }
-= render partial: 'pugin/cards/person-list', locals: { people: @people, letter: params[:letter] }
+.section--primary
+  .container
+    %h1= page_title
+%section
+  .container
+    = render partial: 'pugin/components/letter-navigation', locals: { route_symbol: :people_members_a_z_letter_path }
+%section#content
```
Comment (UsmanAfzal) [inline app/views/people/members_letters.html.haml:9]: Add a space between sections

File: app/views/people/show.html.haml [hunk 7]
```diff
@@ -1,110 +1,106 @@
...
-              %li
-                = "#{l(seat_incumbency.start_date, format: :default)}" unless seat_incumbency.start_date.nil?
-                = seat_incumbency.current? ? t( 'to_present') : " #{t('to')} #{l(seat_incumbency.end_date, format: :default)}"
+      - if @current_party_membership && @current_incumbency
+        .block--border
+          %p.lead= link_to(@current_party_membership.party.name, party_path(@current_party_membership.party.graph_id))
+%section
+  .container
+    - if @current_incumbency && @current_party_membership
+      %section
```
Comment (UsmanAfzal) [inline app/views/people/show.html.haml:58]: Extra section?

**Review by UsmanAfzal — COMMENTED** [review]
Comments added with requested amends

**Pull request merged**